docs/AA--Javascript-in-EE/components/form/select/editor-datetime-select.md

```diff
@@ -0,0 +1,63 @@
+
+# `<EditorDatetimeSelect />`
+
+**EditorDatetimeSelect** applies WordPress admin formatting to the [DatetimeSelect](datetime-select.md) component by wrapping it with the Gutenberg [BaseControl](https://github.com/WordPress/gutenberg/tree/master/components/base-control) component. This wil override the label applied by the **DatetimeSelect** component in favour of the one applied by the **BaseControl** component. Help text can also be specified which will be displayed after the **DatetimeSelect** component.
+
+The **EditorDatetimeSelect** component accepts **all** of the properties that the the **DatetimeSelect** component does as well as the following:
+
+```jsx
+<EditorDatetimeSelect
+    id="datetime-select-1" 
+    className="datetime-select"
+    help={ 'This text will be displayed after the select input to aid the user in understanding its purpose or effect.' }
```

Comment:
Some of this documentation is out-of-date.  Specifically with regards to accepted props.

Reply:
This documentation is still out of date, id is not processed by `EditorDatetimeSelect`